controller/src/main/java/io/pravega/controller/server/eventProcessor/ConcurrentEventProcessor.java

```diff
@@ -40,6 +47,23 @@
 @Slf4j
 public class ConcurrentEventProcessor<R extends ControllerEvent, H extends RequestHandler<R>>
         extends EventProcessor<R> {
+    private static final long RETRY_INITIAL_DELAY = 100;
```

Comment:
I am consolidating all retries within event processors into a single helper class.
However, there are many other places in our code where we do retries. lets consolidate all of them. Opened an issue #1393.